### DISCUSSION TITLE
Restart of ALM calculations

### DISCUSSION THREAD

**Discussion by RicardoCecconello** [ttadano/alamode#218] [Q&A] [answered]
Dear Dr. Tadano,

Is there a way to restart an ALM calculation after the constraints determination? We're calculating a large number of IFCs, and this part takes a lot of time. After this part, it crashes due to memory issues, and we'd like to test this configuration for different DFSETs and conditions of calculation.

**Comment by ttadano** [comment, marked as answer]
Unfortunately, restart mode for ALM is currently not implemented. That option may be useful, so I will consider it as a feature request for the future release.

**Reply by RicardoCecconello** [reply]
Thank you for the consideration!
